exonum-light-client/src/main/java/com/exonum/client/response/TransactionResponse.java

```diff
@@ -73,4 +74,30 @@ public boolean isCommitted() {
     return status == COMMITTED;
   }
 
+  @Override
+  public String toString() {
+    return "TransactionResponse(status=" + this.getStatus()
+        + ", message=" + this.getMessage()
+        + (isCommitted() ? ", executionResult=" + this.getExecutionResult() : "")
+        + ", location=" + this.getLocation() + ")";
+  }
+
+  @Override
+  public boolean equals(Object o) {
```

Comment:
Were these broken too? EqualsVerifier?

Reply:
yeap. Lombok uses getters for equals as well.